### ISSUE TITLE
Export filtered data as a GFF

### ISSUE THREAD

**Issue by SamStudio8** [bag-man/giraffe-browse#2]
Seems like it would be useful to be able to use your tool to subset a GFF and be able to export the derivative table as a new GFF.

**Comment by bag-man** [comment]
How would the data change? Or just simply, selecting n genes to export to a new GFF?

**Comment by SamStudio8** [comment]
Yeah! So I was thinking of a recent use case where I used `grep` to filter out GFF annotations by EC number and then directed them to a new file. Could potentially be useful to be able to do this with your tool? It's low hanging fruit if it ever turns out to be useful :)

**Comment by bag-man** [comment]
That sounds very useful, and totally achievable. Probably won't be in this release but if we take the project further that is a must. 

I can see it being really helpful to be able to just say "I want a GFF with all x,y,z" to run through this other tool; because at the moment it's just a one way process which isn't super useful.
